src/pages/components/subscriptionSection/index.js

```diff
@@ -1,21 +1,28 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 import PlanCard from './PlanCard';
-import { subscriptions } from '../../../utilities/constants';
 
-const SubscriptionSection = () => (
-  <section className='bg-white py-28 px-14 dark:bg-dark-ash-700'>
-    {/* Heading */}
-    <div className='title-heading pb-[80px]'>
-      <h2 className='heading-size dark:text-orange-900'>Boost your Swipet experience</h2>
-    </div>
+const SubscriptionSection = () => {
+  const { t } = useTranslation();
+  const subscriptions = t('subscriptions', { ns: 'features' });
 
-    {/* Cards */}
-    <div className='flex flex-col items-center justify-center gap-y-14 lg:flex-row lg:gap-14 xl:gap-[75px]'>
-      {subscriptions.map((subscription, index) => (
-        <PlanCard subscription={subscription} key={index} />
-      ))}
-    </div>
-  </section>
-);
+  return (
+    <section className='bg-white py-28 px-14 dark:bg-dark-ash-700'>
+      {/* Heading */}
+      <div className='title-heading pb-[80px]'>
+        <h2 className='heading-size dark:text-orange-900'>
+          {t('subscriptionHeading', { ns: 'features' })}
```

Comment:
You can remove the `{ ns: 'features' }`

Please do it to other pages that are applicable :bow: